topic/machine-learning/llm-langchain/requirements.txt

```diff
@@ -1,22 +1,12 @@
-# Real.
 crash
-crate>=1.0.0.dev2
 google-cloud-aiplatform<2
+langchain-cratedb @ git+https://github.com/crate/langchain-cratedb.git@cratedb
```

Comment:
langchain-cratedb 0.0.0 has not been published yet, but it is around the corner.
This patch was about to probe if it's ready.
Apparently, it is.

Reply:
[langchain-cratedb 0.0.0](https://pypi.org/project/langchain-cratedb/) has been released.